nixos/modules/system/activation/activation-script.nix

```diff
@@ -154,9 +154,15 @@ in
 
     system.activationScripts.tmpfs =
       ''
-        ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devSize}" none /dev
```

Comment:
I don't think this is needed, given that during remount only the `size=` should be changing.